docs/datamap/lucene-datamap-guide.md

```diff
@@ -44,12 +44,8 @@ To show all DataMaps created, use:
   ```
 It will show all DataMaps created on main table.
 
-
 ## Lucene DataMap Introduction
-  Lucene is a high performance, full featured text search engine. Lucene is integrated to carbon as
-  an index datamap and managed along with main tables by CarbonData.User can create lucene datamap 
-  to improve query performance on string columns which has content of more length. So, user can 
-  search tokenized word or pattern of it using lucene query on text content.
+  Lucene is a high performance, full featured text search engine. Lucene is integrated to carbon as an index datamap and managed along with main tables by CarbonData.User can create lucene datamap to improve query performance on string columns which has content of more length. So, user can search tokenized word or pattern of it using lucene query on text content.
```

Comment:
is it for string content of more length? or is it natural language sentences stored as string?